buildbot/configure.py

```diff
@@ -134,6 +134,9 @@ def do_configure(args, passthrough_args):
     if args.use_lld:
         llvm_enable_lld = "ON"
 
+    if args.use_zstd:
```

Comment:
do we want to use this in `ci_defaults`?

Reply:
Sure, but let's do that change in a separate PR. I am not sure if the new Windows machine that we recently added (on VM) has `zstd` installed.

Reply:
i should have installed it everywhere but yeah a seperate pr is fine